LR_Tourist/BLL/Services/HotelService.cs

```diff
@@ -0,0 +1,95 @@
+﻿using AutoMapper;
+using BLL.Interfaces;
+using BLL.Model;
+using DA.Data;
+using DA.Services.Repository;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace BLL.Services
+{
+    public class HotelService : IService<HotelDTO>,ICRUDService<HotelDTO>
+    {
+        IRepository<Hotel> repoHotel { get; set; }
```

Comment:
It should be private and readonly and field, not a property.

Reply:
**Refactor all services**

Reply:
fixed